tests/test_admin.py

```diff
@@ -219,53 +222,29 @@ def test_records_filtering_is_generic(self):
 
 class AdminRegisterVersionTestCase(CMSTestCase):
 
-    def setUp(self):
-        self.model = PollVersion
-        self.site = admin.AdminSite()
-
     def test_register_version_admin(self):
-        """Test that calling register_version_admin_for_models registers
-        VersionAdmin for specified models.
-        """
-        register_version_admin_for_models([self.model], self.site)
-
-        self.assertIn(self.model, self.site._registry)
-        self.assertEqual(self.site._registry[self.model].__class__, VersionAdmin)
-
-    def test_register_version_admin_again(self):
-        """Test that, if a version model's admin class is already registered,
-        nothing happens when calling register_version_admin_for_models
-        for that model.
+        """Test that VersionAdmin is registered for Version model
         """
-        register_version_admin_for_models([self.model], self.site)
-
-        with patch.object(self.site, 'register') as mock:
-            register_version_admin_for_models([self.model], self.site)
 
-        mock.assert_not_called()
+        self.assertIn(Version, admin.site._registry)
+        self.assertEqual(admin.site._registry[Version].__class__, VersionAdmin)
 
 
 class VersionAdminTestCase(CMSTestCase):
 
     def setUp(self):
-        self.model = PollVersion
+        self.model = Version
         self.site = admin.AdminSite()
-        register_version_admin_for_models([self.model], self.site)
+        self.site.register(Version, VersionAdmin)
         self.superuser = self.get_superuser()
 
-    def test_admin_queryset_num_queries(self):
-        """Test that accessing Version.content in changelist
-        doesn't result in additional query
-        """
-        self.assertIn(
-            'content',
-            self.site._registry[PollVersion].list_select_related,
-        )
-
     def test_content_link(self):
-        version = factories.PollContentWithVersionFactory(text='test4').pollversion
+        pc = factories.PollContentWithVersionFactory(text='test4')
+        version = self.model.objects.get(
+            content_type=ContentType.objects.get_for_model(pc),
+            object_id=pc.pk)
```

Comment:
Not particularly important, but I think atm you could just do `version = factories.PollVersionFactory(content__text='test4')` and then refer to content as `version.content`. It would save some lines and be a little easier to read probably.